### ISSUE TITLE
grid nx & ny are not doing anything

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/gsplot#383]
The arguments `nx` and `ny` in `grid` are not actually being used. Also, the grid lines are plotting overtop of the box.

``` r
gs <- gsplot() %>% 
    points(1:10,1:10) %>% 
    grid(nx=NA, ny=2)
gs

Warning messages:
1: In int_abline(a = a, b = b, h = h, v = v, untf = untf, ...) :
  "nx" is not a graphical parameter
2: In int_abline(a = a, b = b, h = h, v = v, untf = untf, ...) :
  "ny" is not a graphical parameter
```

![image](https://cloud.githubusercontent.com/assets/13220910/16631327/3c069d98-4384-11e6-83b6-83adb4ab3c7d.png)

**Comment by jordansread** [comment]
@lindsaycarr is this a deal breaker for stable release?

**Comment by lindsayplatt** [comment]
I've got a possible fix coming through on branch right now, but yes - repgen uses it a lot

**Comment by jordansread** [comment]
perhaps related to https://github.com/USGS-R/gsplot/issues/343 ?

**Comment by lindsayplatt** [comment]
Possibly? But I don't think that covers all the issues (maybe the grid lines over the box). The arguments for ny & nx were not being passed along or used at all. Checkout the graphics::grid function compared to what we have https://github.com/USGS-R/gsplot/blob/master/R/grid.R#L56-L77

**Comment by lindsayplatt** [comment]
See #384 for a possible solution

**Comment by jordansread** [comment]
fixed in #386

**Comment by lindsayplatt** [comment]
@jread-usgs did not close yet because of the box issue. The grid lines are covering the box around the plot

**Comment by jordansread** [comment]
Oh, missed that. ordering of `print`?

**Comment by lindsayplatt** [comment]
Yeah, I think so - looking into it now

**Comment by lindsayplatt** [comment]
grid lines covering box and tick marks fixed in #390
